crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -183,6 +193,122 @@ pub struct ClientReferenceInformation {
     code: Option<String>,
 }
 
+impl
+    TryFrom<(
+        &BankOfAmericaRouterData<&types::PaymentsAuthorizeRouterData>,
+        payments::Card,
+    )> for BankOfAmericaPaymentsRequest
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        value: (
+            &BankOfAmericaRouterData<&types::PaymentsAuthorizeRouterData>,
+            payments::Card,
+        ),
+    ) -> Result<Self, Self::Error> {
+        let (item, ccard) = value;
+        let email = item.router_data.request.get_email()?;
+        let bill_to = build_bill_to(item.router_data.get_billing()?, email)?;
+
+        let order_information = OrderInformationWithBill {
```

Comment:
Handle this with From impl so redundant code can be reduced

Reply:
Done